tests/e2e/core-tests/specs/shopper/front-end-checkout-login-account.test.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable jest/no-export, jest/no-disabled-tests, jest/expect-expect */
+/**
+ * Internal dependencies
+ */
+ const {
+	shopper,
+	merchant,
+	createSimpleProduct,
+	uiUnblocked,
+	setCheckbox,
+	settingsPageSaveChanges,
+} = require( '@woocommerce/e2e-utils' );
+
+/**
+ * External dependencies
+ */
+const {
+	it,
+	describe,
+	beforeAll,
+} = require( '@jest/globals' );
+
+const config = require('config');
+const simpleProductName = config.get('products.simple.name');
+
+const runCheckoutLoginAccountTest = () => {
+	describe('Shopper Checkout Login Account', () => {
+		beforeAll(async () => {
+			await merchant.login();
+			await createSimpleProduct();
+
+			// Set checkbox for logging to account during checkout
+			await merchant.openSettings('account');
+			await setCheckbox('#woocommerce_enable_checkout_login_reminder');
+			await settingsPageSaveChanges();
+			await merchant.logout();
+
+			// Add simple product to cart and proceed to checkout
+			await shopper.goToShop();
+			await shopper.addToCartFromShopPage(simpleProductName);
+			await uiUnblocked();
+			await shopper.goToCheckout();
+		});
+
+		it('can login to an existing account during checkout', async () => {
+			// Click to login during checkout
+			await page.waitForSelector('.woocommerce-form-login-toggle');
+			await expect(page).toClick('.woocommerce-info > a.showlogin');
+
+			// Fill shopper's login credentials and proceed further
+			await page.type( '#username', config.get('users.customer.username') );
+			await page.type( '#password', config.get('users.customer.password') );
+
+			await Promise.all([
```

Comment:
Looks like the spacing on this is a little off, we just need to adjust the tabbing a bit, like below:

```diff
-
	await Promise.all([
	page.waitForNavigation({waitUntil: 'networkidle0'}),
	page.click('button[name="login"]'),
]);


+
await Promise.all([
	page.waitForNavigation({waitUntil: 'networkidle0'}),
	page.click('button[name="login"]'),
]);
```

We also want to make sure the closing `]);` is in line with the opening `await` in the `await Promise.all(` line.

Reply:
You were right, will update the code 👍

Reply:
Thanks!